### PR TITLE
Print reportTCB and firmware version

### PR DESCRIPTION
**Pull request by LiuJohn-1886**
- Printing TCB, firmware, and TEE information is a critical step in the SNPCVM attestation process. By verifying these key pieces of information, you can ensure that all components of the system are in the expected secure state.
- This code change has been verified on a SNPCVM.

### CODE REVIEW

File: cvm-attestation/src/ImdsClient.py [hunk 1]
```diff
@@ -103,6 +103,7 @@ def get_vcek_certificate(self):
         if response.status_code == 200:
           self.log.info("Received certificate successfully")
           data_json = json.loads(response.text)
+          self.log.info(f"Reported TCB: {data_json['tcbm']}")
```
Comment (lizzha) [inline cvm-attestation/src/ImdsClient.py:106]: could you make it more specific to tell it's the TCB version of the Certificate? maybe you can update the log 104, "Received certificate successfully, TCB version: XXX"

File: cvm-attestation/src/AttestationProvider.py [hunk 2]
```diff
@@ -194,6 +194,7 @@ def print_snp_platform_claims(self, encoded_token):
         self.log.info(f"Bootloader SVN: {claims['x-ms-sevsnpvm-bootloader-svn']}")
         self.log.info(f"Guest SVN: {claims['x-ms-sevsnpvm-guestsvn']}")
         self.log.info(f"Microcode SVN: {claims['x-ms-sevsnpvm-microcode-svn']}")
+        self.log.info(f"Firmware SVN: {claims['x-ms-sevsnpvm-snpfw-svn']}")
```
Comment (lizzha) [inline cvm-attestation/src/AttestationProvider.py:197]: "SNP Firmware SVN"?
Could you output the TEE SVN as well?

Comment (jvega10) [inline cvm-attestation/src/AttestationProvider.py:197]:
```suggestion
        self.log.info(f"SNP Firmware SVN: {claims['x-ms-sevsnpvm-snpfw-svn']}")

```

Comment (LiuJohn-1886) [reply cvm-attestation/src/AttestationProvider.py]: Make a change as your comment. Thanks!

**Review by lizzha — APPROVED** [review]

**Pull request merged**